Lib/fontTools/colorLib/builder.py

```diff
@@ -526,7 +527,20 @@ def buildPaintRadialGradient(
         ot_paint.Format = int(ot.Paint.Format.PaintRadialGradient)
         ot_paint.ColorLine = _to_color_line(colorLine)
 
-        for i, (x, y), r in [(0, c0, r0), (1, c1, r1)]:
+        # normalize input types (which may or may not specify a varIdx)
+        x0, y0 = _to_variable_value(c0[0]), _to_variable_value(c0[1])
+        r0 = _to_variable_value(r0)
+        x1, y1 = _to_variable_value(c1[0]), _to_variable_value(c1[1])
+        r1 = _to_variable_value(r1)
+
+        # avoid abrupt change after rounding when c0 is near c1's perimeter
```

Comment:
If acceptable in FT # <link to gh issue> would be nice

Reply:
pardon?

Reply:
oh now I read your comment (the markdown formatting somehow broke it). Yes, I did add a link, its in the docstring of `nudge_start_circle_almost_inside`